docs/content/en/docs/tasks/packages/prereq.md

```diff
@@ -0,0 +1,33 @@
+## Prerequisites
+Before installing any curated packages for EKS Anywhere, do the following:
+
+* Check that the version of `eksctl anywhere` is `v0.9.0` or above with the following commands:
+    ```bash
+    eksctl anywhere version
+    ```
+    Example command output
+    ```
+    NAME                                       READY   STATUS     RESTARTS   AGE
+    eks-anywhere-packages-57778bc88f-587tq     2/2     Running    0          16h
+    ```
+* Make sure cert-manager is up and running in the cluster. Note cert-manager is not installed on workload clusters by default. If cert-manager is not installed, you can manually install cert-manager and follow the instructions below to finish the package controller installation.
+
+* Check the existence of package controller:
+
+    ```bash
+    kubectl get pods -n eksa-packages | grep "eks-anywhere-packages"
+    ```
+    If the returned result is empty, you need to install the package controller.
+
+* If the packages controller is not yet installed, follow these steps:
+
+    1. Install the package controller
+        ```bash
+        export CURATED_PACKAGES_SUPPORT=true
+        eksctl anywhere install packagecontroller -f $CLUSTER_NAME.yaml
```

Comment:
You're missing a step where you add the package controller to the cluster config.

Reply:
This command is going to be deprecated or maybe it is already too.

Reply:
As I noted above, we should decide if we want to update this PR to reflect the new way of setting up the package controller or just close it and add the content separately. Either way, I think having it in a separate prereq file that's read into each task is a good idea.